posthog/temporal/batch_exports/squash_person_overrides.py

```diff
@@ -51,7 +51,7 @@
     dictHas('{database}.{dictionary_name}', (team_id, distinct_id))
     {in_team_ids}
 SETTINGS
-    max_execution_time = 0
+    max_execution_time = 86400
```

Comment:
The problem is that mutations happen in the background because default is to run them async. You need to set this setting as well:

https://clickhouse.com/docs/en/operations/settings/settings#mutations_sync

```suggestion
    max_execution_time = 86400,
    mutations_sync = 2
```